### ISSUE TITLE
volk_8ic_x2_multiply_conjugate_16ic is not documented

### ISSUE THREAD

**Issue by argilo** [gnuradio/volk#697]
The `volk_8ic_x2_multiply_conjugate_16ic` kernel does not have a documentation page.

**Comment by gitty-prakhar** [comment]
Hi I'd like to solve this issue, is it still relevant?

**Comment by argilo** [comment]
Yes, it is still the case that `volk_8ic_x2_multiply_conjugate_16ic` lacks a documentation page.
